cmd/aws-cloud-controller-manager/controllers/tagging/tagging-controller.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tagging
+
+import (
+	"k8s.io/klog/v2"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+// TaggingController is the controller implementation for tagging cluster resources
+type TaggingController struct {
+}
+
+// NewTaggingController creates a NewTaggingController object
+func NewTaggingController() (*TaggingController, error) {
+	tc := &TaggingController{
```

Comment:
Should this controller take in the list of tags as inputs?

Reply:
yes, that's something that I'm considering next. As for a skeleton, it doesn't take in anything at the moment.